libp2p/Host.cpp

```diff
@@ -825,10 +842,12 @@ void Host::startedWorking()
     else
         m_listenPort = m_netConfig.listenPort;
 
-    determinePublic();
+    m_tcpPublic = determinePublic();
```

Comment:
I made `determinePublic` return `tcp::endoiunt` as result, instead of assigning it inside, to make more explicit that it's the only result of this function.